### ISSUE TITLE
Create 'batch change' to replace deprecated io/ioutil package

### ISSUE THREAD

**Issue by daxmc99** [sourcegraph/sourcegraph#18739]
The `io/ioutil` package was deprecated as part of Go 1.16.

The **batch change** should remove all usage of `io/ioutil` and replace it with the correct usage of the `io` package.

**Comment by ryanslade** [comment]
Details are here:
https://golang.org/doc/go1.16#ioutil

**Comment by ryanslade** [comment]
I've started having a go at this while I wait for integration tests to build.

**Comment by ryanslade** [comment]
I think this should do it: https://k8s.sgdev.org/users/rslade/batch-changes/apply/QmF0Y2hTcGVjOiIxQjQ4RTdjUkRDZSI=

**Comment by ryanslade** [comment]
I've closed that change as it doesn't appear to hit every match and goimports didn't always do the right thing.

I've attached the batch spec for reference and will take another look when I'm back on Tuesday.

[deprecate-ioutil.txt](https://github.com/sourcegraph/sourcegraph/files/6238055/deprecate-ioutil.txt)

It seems that comby is missing some replacement. I ran the comby command in the spec locally on out main repo and then looked for uses of `ioutil` with ripgrep and there were still many left over.
